git clone via https protocol instead of ssh

iamoskvin opened the pull request:
Validators can't clone using their ssh keys.

ccamel commented:
@iamoskvin Thanks for your PR!

If you don't mind, can you please:
- rebase your work against `main` (following changes on the `main` branch, your PR is now in conflict),
- fix the commit message to respect the conventional commit formalism (see [okp4/wiki/Naming-rules#commit-messages](https://github.com/okp4/wiki/wiki/Naming-rules#commit-messages)). You can prefix with `fix:`.

Many thanks. 😌

iamoskvin commented:
> @iamoskvin Thanks for your PR!
> 
> If you don't mind, can you please:
> 
> * rebase your work against `main` (following changes on the `main` branch, your PR is now in conflict),
> * fix the commit message to respect the conventional commit formalism (see [okp4/wiki/Naming-rules#commit-messages](https://github.com/okp4/wiki/wiki/Naming-rules#commit-messages)). You can prefix with `fix:`.
> 
> Many thanks. relieved

Thank you.
I have fixed the commit message (git commit --amend). 
Also, I tried to rebase but I suffer from conflicts between main branch and other people commits at this PR branch (file docs/index rename, I think). Unfortunately, I am not a git profi, so I don't know how to resolve this problem...

"git pull --rebase upstream main
remote: Enumerating objects: 45, done.
remote: Counting objects: 100% (45/45), done.
remote: Compressing objects: 100% (30/30), done.
remote: Total 45 (delta 25), reused 31 (delta 15), pack-reused 0
Unpacking objects: 100% (45/45), done.
From https://github.com/okp4/docs
 * branch            main       -> FETCH_HEAD
 * [new branch]      main       -> upstream/main
First, rewinding head to replay your work on top of it...
Applying: feat: correct grammar errors in abstract
Applying: feat: change title of index
Using index info to reconstruct a base tree...
A	docs/index
.git/rebase-apply/patch:10: trailing whitespace.
# OKP4 Whitepaper and Documentation
warning: 1 line adds whitespace errors.
Falling back to patching base and 3-way merge...
CONFLICT (modify/delete): docs/index deleted in 28d6276c8c771bb67005c36d38f9062e3327fe17 and modified in feat: change title of index. Version feat: change title of index of docs/index left in tree.
error: Failed to merge in the changes.
Patch failed at 0002 feat: change title of index
The copy of the patch that failed is found in: .git/rebase-apply/patch"

ccamel commented:
@iamoskvin I see. We had to push on the principal and that's why your PR is now of a conflict. Sorry for that.

To solve the issue, I would like to suggest the following approach (by re-integrating https://github.com/okp4/docs/commit/774adb89fc8017de2549a0c770ce8ef5f13d0a98)

on your branch:
```
git reset --hard upstream/main
git cherry-pick 774adb8
git commit --amend --message "fix: git clone via https protocol instead of ssh"
git push --force
```

Tell me if it works. Thanks to you.

iamoskvin commented:
> @iamoskvin I see. We had to push on the main branch and that's why your PR is now conflicting. Sorry for that.
> 
> To solve the issue, I would like to suggest the following approach (by re-integrating [774adb8](https://github.com/okp4/docs/commit/774adb89fc8017de2549a0c770ce8ef5f13d0a98))
> 
> on your branch:
> 
> ```
> git reset --hard upstream/main
> git cherry-pick 774adb8
> git commit --amend --message "fix: git clone via https protocol instead of ssh"
> git push --force
> ```
> 
> Tell me if it works. Thanks to you.

Many thanks for your assistance! 
I think it worked for me. I have picked another commit from my repo (I can't pick 774adb8).
Please check the result.

ccamel commented:
@amimart ok for you?

ccamel commented:
Great. Thanks :+1:
